pymisp/abstract.py

```diff
@@ -102,17 +155,32 @@ def __init__(self, **kwargs):
             setattr(AbstractMISP, 'add_tag', AbstractMISP.__add_tag)
             setattr(AbstractMISP, 'tags', property(AbstractMISP.__get_tags, AbstractMISP.__set_tags))
 
+    @property
+    def describe_types(self):
+        return self.__describe_types
+
+    @describe_types.setter
+    def describe_types(self, describe_types):
+        self.__describe_types = describe_types
+
+    @property
+    def resources_path(self):
+        return self.__resources_path
+
+    @property
+    def misp_objects_path(self):
+        return self.__misp_objects_path
+
+    @misp_objects_path.setter
+    def misp_objects_path(self, misp_objects_path):
+        self.__misp_objects_path = misp_objects_path
```

Comment:
I don't see this property being set anywhere. Is this existing public API or could this be removed?

Reply:
Good catch, I wanted to normaize misp_objects_path_custom with it, and forgot. working on that now.